tests/engine/test_arg_utils.py

```diff
@@ -71,6 +72,21 @@ def test_get_type(type_hints, type, expected):
     assert get_type(type_hints, type) == expected
 
 
+@pytest.mark.parametrize(("type_hints", "expected"), [
+    ({Literal[1, 2]}, {
+        "type": int,
+        "choices": [1, 2]
+    }),
+    ({Literal[1, "a"]}, Exception),
+])
+def test_literal_to_kwargs(type_hints, expected):
+    context = nullcontext()
+    if expected is Exception:
```

Comment:
minor thing, not important ...

```suggestion
    if isinstance(expected, Exception):
```

Reply:
This seems to be still here